MultiWaveform waveform
---
bool success
string status_message
